### PR TITLE
Fix JSON path to overview docs

### LINKED ISSUE #366: [CT-1980] Welcome message not shown in dbt docs
### Describe the bug
The overview page of dbt docs shows "(loading)" instead of the welcome message. There is an error in the console: "TypeError: Cannot read properties of undefined (reading 'block_contents')"

### Steps To Reproduce
Run `dbt init`, `dbt docs generate` and `dbt docs serve` with dbt version 1.4.

### Expected behavior
Expected to see the default welcome message on the overview page of dbt docs.

### Screenshots and log output
image

**The output of `dbt --version`:**
```
Core:
  - installed: 1.4.1
  - latest:    1.4.1 - Up to date!

Plugins:
  - bigquery: 1.4.0 - Up to date!
```

### PR DESCRIPTION
**Pull request by halvorlu**
Resolves #366

### Description
A minor change to correct the path to the overview markdown in `manifest.json`

### Checklist

- [x] I have read [the contributing guide](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md) and understand what's expected of me
- [x] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [x] I have run this code in development and it appears to resolve the stated issue
- [x] I have [opened an issue to add/update docs](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose), or docs changes are not required/relevant for this PR
- [x] I have run `changie new` to [create a changelog entry](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md#Adding-CHANGELOG-Entry)

### CODE REVIEW

**Comment by halvorlu** [comment]
I'm not sure if I'm supposed to run `changie` or not, please advise :)

**Comment by dbeatty10** [comment]
Thanks for discovering the issue and opening this PR!

> I'm not sure if I'm supposed to run `changie` or not, please advise :)

Yeah, do `changie new`!

When we create the next release, it _should_* be picked up and added to the changelog in dbt-core like [this](https://github.com/dbt-labs/dbt-core/blob/1.4.latest/CHANGELOG.md#docs).

(*We'll plan on including this as a backport to a 1.4.X release, and I'm not sure how exactly backports affect changie+changelogs.)

**Comment by emmyoop** [comment]
@halvorlu thanks for the fix on this!

I've tested that this resolves the bug reported and custom overviews also continue to work as expected.  Once you add your changelog entry, I can merge this in to get it a part of the next patch release for 1.4.

**Comment by dbeatty10** [comment]
@emmyoop I just added a changelog entry. Does this unblock this PR to move forward?

**Review by emmyoop — APPROVED** [review]

**Pull request merged**
